backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -14,7 +14,7 @@ const DEFAULT_IDLENESS_CHECK_PERIOD = '1'; // 1 minute
 const DEFAULT_CLUSTER_SETTINGS: ClusterSettings = {
   pvcSize: DEFAULT_PVC_SIZE,
   cullerTimeout: DEFAULT_CULLER_TIMEOUT,
-  userTrackingEnabled: null,
+  userTrackingEnabled: false,
 };
 
 export const updateClusterSettings = async (
```

Comment:
This method is a bit of a mess. Things you need to change:

1. Avoid adding more try/catches for single API calls... just do a `.catch` on the end and let it flow through to the next call
2. Avoid throwing errors when it doesn't matter if the call succeeded; if the call is not critical to the success of the API call, don't break threads
3. If you do not first check if the K8s resource exists, doing patches/deletes and the likewise will be counter productive -- look to follow step no1 and just do `.catch` and suppress those failures
4. If you're going to catch for 404, don't forget to throw the error -- or return -- if it's NOT 404 -- see the bottom catch of this method, it just defaults to `undefined` being returned from this function; which Fastify does not like and it breaks the API call from the client

If we were not moving away from the server with the pass-through API, I'd request this function be broken up into several other functions so we can do proper separation of concerns. The saving of this page saves multiple items and it's hard to follow which is which inside this function -- makes it harder to maintain and non-related items can cause problems (which is what the original bug noted).